galsim/wfirst/wfirst_detectors.py

```diff
@@ -82,7 +82,28 @@ def applyIPC(img, edge_treatment='extend', fill_value=None):
     """
     img.applyIPC(galsim.wfirst.ipc_kernel, edge_treatment=edge_treatment, fill_value=fill_value)
 
-def allDetectorEffects(img, rng=None, exptime=None):
+def applyPersistence(img, prev_exposures):
+    """
+    Applies the persistence effect to the Image instance by adding a small fraction of the previous
+    exposures (up to 8) supplied as the 'prev_exposures' argument.
```

Comment:
Should probably mention which version of the image should be provided here.  Is it the final image in ADU after readout?  Or the image of electron counts?  With or without IPC?  etc.

From where you placed it in the `allDetectorEffects` function, it should include reciprocity failure, but not IPC or non-linear response.  This subtlety should be (a) verified, but I assume you have this correct, and (b) described here in the doc string.